js/app/pages/Archive.html

```diff
@@ -191,6 +191,25 @@
         }
 
         // filter by errors
+        // filter by incomplete forms
```

Comment:
The reporting fine tuning can come in a later PR - nice though to include initially.  Good thinking.
My preference would be for the two cases to be separate checkboxes: "students left without checking out" and "items missing".  The descriptions should be very clear on what they represent.  "incomplete" is very general and not self explanatory - the user may expect to see something different; at the least, we have to explain to new users what it does, rather than let the description speak for itself

Reply:
I agree with this. Would you like me to remove the option from the html entirely or change it to the separate checkboxes in another commit

Reply:
a separate PR would be cleaner, but only if its not much trouble.

Reply:
I'll leave it as is for a placeholder and then make a PR with the new behavior.